cmd/controller/main.go

```diff
@@ -25,47 +25,64 @@ import (
 var version = "dev"
 var name = "deviceplane-controller"
 
+//commit for test
 var (
 	addr = kingpin.
 		Flag("addr", "").
 		Default(":8080").
+		OverrideDefaultFromEnvar("ADDR").
 		String()
 	mysql = kingpin.
 		Flag("mysql", "").
 		Default("deviceplane:deviceplane@tcp(localhost:3306)/deviceplane?parseTime=true").
+		OverrideDefaultFromEnvar("MYSQL_CONNECTION_STRING").
 		String()
+	enableSignUp = kingpin.
+			Flag("enableSignUp", "").
+			OverrideDefaultFromEnvar("ENABLE_SING_UP").
+			Bool()
 	statsdAddress = kingpin.
 			Flag("statsd", "").
 			Default("127.0.0.1:8125").
+			OverrideDefaultFromEnvar("STATS_ADDRESS").
```

Comment:
Same concern as above about keeping the env var name and the flag name as similar as possible, should this be `STATSD`?